app/templates/src/index.adoc

```diff
@@ -0,0 +1,58 @@
+= <%= title %>
+:!sectids:
+:imagesdir: images
+:source-highlighter: coderay
+:coderay-css: style
+
+== Images
+image::bespoke-logo.jpg[]
+
+== Keyboard Interaction
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-keys[bespoke-keys]
+
+== Touch Interaction
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-scale[bespoke-scale]
+
+== Bullet List
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-bullets[bespoke-bullets]
+[.build.build-items]
+* Bullet 1
+* Bullet 2
+* Bullet 3
+
+<% if (usePluginMultimedia) { -%>
+== !
+video::AV3kYPutYfs[youtube,640,360,opts="loop,muted"]
+<% } -%>
+
+== Syntax Highlighting
+[source,js]
+----
+(function() {
+  function hanoi(n, a, b, c) {
+    return n ? hanoi(n - 1, a, c, b)
+        .concat([[a, b]])
+        .concat(hanoi(n - 1, c, b, a)) : [];
+  }
+  return hanoi(3, 'left', 'right', 'mid')
+    .map(function(d) {
+      return d[0] + ' -> ' + d[1];
+    });
+})();
+----
+
+// No named route plugin usage?
+
+== Just the beginning...
+
+[.build.build-items]
```

Comment:
Same here. We should keep this as `%build`. When I use `%build` on the list and `.build` on the paragraph that follows, the paragraph shows up after all the other items have built.